packages/consumption/src/consumption/CoreErrors.ts

```diff
@@ -150,6 +150,10 @@ class Attributes {
         );
     }
 
+    public cannotSucceedAttributesWithDeletionInfo() {
+        return new CoreError("error.consumption.attributes.cannotSucceedAttributesWithDeletionInfo", "You cannot succeed attributes with a deletionInfo.");
+    }
```

Comment:
I think the error and description doesn't actually show the user what is going wrong (the user doesn't necessarily know about a deletionInfo, just about the deletion itself), so we should just inform the user about the deletion itself but not about the deletionInfo

Reply:
The deletionInfo states that the Attribute may have been deleted by the peer of the Relationship or it may be marked for deletion by yourself or by the owner. So an Attribute may have a deletionInfo, even though no deletion took place, yet. Therefore, I think it is most accurate to use the term deletionInfo. But, I can extent the error message.

Reply:
I'd prefer the technical error message since we're in the consumption library, i.e. _we_ are the user.

Reply:
we push this to the runtime and from the runtime to the connector so NO, we aren't :)

Reply:
> But, I can extent the error message.
yep, please make sure what the deletion info is about and why you can't do this

Reply:
> we push this to the runtime and from the runtime to the connector so NO, we aren't :)

Well, _if_ we do this, fair enough.